src/locale/id.js

```diff
@@ -0,0 +1,43 @@
+import dayjs from 'dayjs'
+
+const locale = {
+  name: 'id',
+  MS: 'milidetik',
+  S: 'detik',
+  MIN: 'menit',
+  H: 'jam',
+  D: 'hari',
+  W: 'minggu',
+  M: 'bulan',
+  Q: 'kuartal',
+  Y: 'tahun',
+  DATE: 'tanggal',
+  weekdays: 'Minggu_Senin_Selasa_Rabu_Kamis_Jum\'at_Sabtu'.split('_'),
```

Comment:
should this be Jumat ? I don't know, just feel weird

Reply:
Hmmm okey i'll fix it